.gitignore

```diff
@@ -97,3 +97,5 @@ config/secrets.*.json
 # macOS
 
 .DS_Store
+
+Configurations/TeamID.xcconfig
```

Comment:
What if we had a third build configuration for external contributions (inheriting from Debug settings), that referenced a separate ignored `xcconfig` file?

Reply:
There is no way (that I know of) to conditionally include an xcconfig file if it exists. So this would end up causing a build error and you'd have to create the file manually (or via a script)

Reply:
That's kind of what I was thinking.  Would it be okay to produce an error until the file is committed?  That would only need to be done once, right?

I'm open to ideas but I'd like to prioritize a no-setup experience for internal contributors.